core/integration/features/verified/0042-LIQF-SLA.feature

```diff
@@ -0,0 +1,771 @@
+      | aux2  | ETH/DEC23 | sell | 1      | 1010  | 0                | TYPE_LIMIT | TIF_GTC |
+    And the opening auction period ends for market "ETH/DEC23"
+    Then the trading mode should be "TRADING_MODE_CONTINUOUS" for the market "ETH/DEC23"
+
+    # Generate liquidity fees to be allocated to the LP
+    Given the parties place the following orders:
+      | party  | market id | side | volume | price | resulting trades | type       | tif     |
+      | party1 | ETH/DEC23 | buy  | 2      | 1000  | 0                | TYPE_LIMIT | TIF_GTC |
+      | party2 | ETH/DEC23 | sell | 2      | 1000  | 1                | TYPE_LIMIT | TIF_GTC |
+    Then the accumulated liquidity fees should be "200" for the market "ETH/DEC23"
+
+    # Ensure LP1s average timeBookFraction ~0.75 and penalties will be ~0.5 in the next epoch
+    Given the network moves ahead "90" blocks
+    And the parties cancel the following orders:
+      | party | reference      |
+      | lp1   | lp1-ice-buy-1  |
+      | lp1   | lp1-ice-sell-1 |
+    # Ensure LP2s average timeBookFraction ~0.875 and penalties will be ~0.75 in the next epoch
+    Given the network moves ahead "15" blocks
+    And the parties cancel the following orders:
+      | party | reference      |
+      | lp2   | lp2-ice-buy-1  |
+      | lp2   | lp2-ice-sell-1 |
+    And the network moves ahead "1" epochs
+    Then the following transfers should happen:
+      | from | to  | from account                                   | to account                                     | market id | amount | asset |
+      |      | lp1 | ACCOUNT_TYPE_FEES_LIQUIDITY                    | ACCOUNT_TYPE_LP_LIQUIDITY_FEES                 | ETH/DEC23 | 100    | USD   |
+      |      | lp2 | ACCOUNT_TYPE_FEES_LIQUIDITY                    | ACCOUNT_TYPE_LP_LIQUIDITY_FEES                 | ETH/DEC23 | 100    | USD   |
+      | lp1  | lp1 | ACCOUNT_TYPE_LP_LIQUIDITY_FEES                 | ACCOUNT_TYPE_GENERAL                           | ETH/DEC23 | 50     | USD   |
+      | lp1  |     | ACCOUNT_TYPE_LP_LIQUIDITY_FEES                 | ACCOUNT_TYPE_LIQUIDITY_FEES_BONUS_DISTRIBUTION | ETH/DEC23 | 50     | USD   |
+      | lp2  | lp2 | ACCOUNT_TYPE_LP_LIQUIDITY_FEES                 | ACCOUNT_TYPE_GENERAL                           | ETH/DEC23 | 75     | USD   |
+      | lp2  |     | ACCOUNT_TYPE_LP_LIQUIDITY_FEES                 | ACCOUNT_TYPE_LIQUIDITY_FEES_BONUS_DISTRIBUTION | ETH/DEC23 | 25     | USD   |
+      |      | lp1 | ACCOUNT_TYPE_LIQUIDITY_FEES_BONUS_DISTRIBUTION | ACCOUNT_TYPE_GENERAL                           | ETH/DEC23 | 30     | USD   |
+      |      | lp2 | ACCOUNT_TYPE_LIQUIDITY_FEES_BONUS_DISTRIBUTION | ACCOUNT_TYPE_GENERAL                           | ETH/DEC23 | 45     | USD   |
+
+
+  Scenario: 2 LPs fulfill the mininum time fraction but have different SLA performance when the sla competition factor is non-zero. (0042-LIQF-043)(0042-LIQF-044)
```

Comment:
Are the last two scenarios exactly the same? Do we need either of them if same ACs are already covered by previous scenarios?

Reply:
Thanks @witgaw yeah thats a duplicate test, AC incorrect on the last one as well. Fixed now.